guides/code-style/lints.rake

```diff
@@ -0,0 +1,16 @@
+require 'rubocop/rake_task'
+
+RuboCop::RakeTask.new
+
+task :default => :rubocop
```

Comment:
Rake files are excluded from hash syntax checks.

Reasoning behind this is that Rake used hashrocket syntax to denote dependencies.